### PR TITLE
Update half of the test websites to use new Hosting API conventions.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
https://github.com/aspnet/KestrelHttpServer/pull/496

### CODE REVIEW

File: Mvc.sln [hunk 1]
```diff
@@ -100,8 +100,6 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.AspNet.Mvc.Abstra
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "MvcMinimalSample.Web", "samples\MvcMinimalSample.Web\MvcMinimalSample.Web.xproj", "{F21E225B-190B-4DAA-8B0A-05986D231F56}"
 EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "AntiforgeryTokenWebSite", "test\WebSites\AntiforgeryTokenWebSite\AntiforgeryTokenWebSite.xproj", "{A353B17E-A940-4CE8-8BF9-179E24A9041F}"
```
Comment (pranavkm) [inline Mvc.sln]: Did this get removed?

File: test/WebSites/ApiExplorerWebSite/project.json [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+  },
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.AspNet.Server.Kestrel": "1.0.0-*",
+    "Microsoft.AspNet.Mvc": "6.0.0-*",
+    "Microsoft.AspNet.Mvc.Formatters.Xml": "6.0.0-*",
+    "Microsoft.AspNet.Mvc.TestConfiguration": "1.0.0",
+    "Microsoft.AspNet.Server.WebListener": "1.0.0-*",
```
Comment (pranavkm) [inline test/WebSites/ApiExplorerWebSite/project.json]: Remove (also I've been moving Kestrel down here so it's alphabetic). Second bit is totally optional.

Comment (NTaylorMullen) [inline test/WebSites/ApiExplorerWebSite/project.json]: :+1:

File: Mvc.sln (hunk 1 above)
Comment (NTaylorMullen) [inline Mvc.sln]: Yup

**Comment by NTaylorMullen** [comment]
:up::date:

**Comment by pranavkm** [comment]
:shipit:

File: test/WebSites/ApiExplorerWebSite/project.json [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
+    "Microsoft.AspNet.Mvc": "6.0.0-*",
+    "Microsoft.AspNet.Mvc.Formatters.Xml": "6.0.0-*",
+    "Microsoft.AspNet.Mvc.TestConfiguration": "1.0.0",
+    "Microsoft.AspNet.StaticFiles": "1.0.0-*"
+  },
+  "frameworks": {
+    "dnx451": { },
+    "dnxcore50": { }
+  },
+  "webroot": "wwwroot"
```
Comment (JunTaoLuo) [inline test/WebSites/ApiExplorerWebSite/project.json]: remove webroot, do this in all project file

**Pull request merged**